crates/agglayer-prover/src/lib.rs

```diff
@@ -52,11 +51,6 @@ pub fn main(cfg: PathBuf, version: &str, program: &'static [u8]) -> anyhow::Resu
     Ok(())
 }
 
-pub fn get_vkey(program: &'static [u8]) -> String {
-    let vkey = prover_executor::Executor::get_vkey(program);
-    vkey.bytes32()
-}
-
```

Comment:
Why this public function is removed? [It is used here](https://github.com/agglayer/agglayer/blob/main/crates/agglayer/src/main.rs#L42)

Reply:
I would prefer to keep the logic encapsulated into the `prover` crate, the CLI shouldn't have to have `sp1-sdk` or even `prover-executor` as direct dep.
I'm fine refactoring this to a standalone function, tho. Something like:

```rust

// prover_executor crate
pub fn get_program_vkey(program: &'static [u8]) -> String { ... }

// agglayer_prover

pub fn get_vkey(program: &'static [u8]) -> String {
    let vkey = prover_executor::get_program_vkey(program);
    vkey.bytes32()
}
```